### DISCUSSION TITLE
To achieve the function of the piecewise function, there is no class that can be used, or a custom equation class

### DISCUSSION THREAD

**Discussion by Mrxiongjy** [idaholab/moose#27656] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

<Your question here>
How to implement the function of the segmentation function, is there a class that can be used, or a custom equation class

```
[bu]
type = ParsedFunction
Expression (ti ^ 2) = 1.25 * * 270 / (10431 * 235 * 3.1416 * (4.1 * 10 ^ 3) ^ 2) + 0.001
symbol_names = 'ti'
symbol_values = 'time'
[]
```

**Comment by Mrxiongjy** [comment]
How to implement the function of the segmentation function, is there a class that can be used, or a custom equation class
```
[bu]
type = ParsedFunction
Expression (ti ^ 2) = 1.25 * * 270 / (10431 * 235 * 3.1416 * (4.1 * 10 ^ 3) ^ 2) + 0.001
symbol_names = 'ti'
symbol_values = 'time'
[]
```

**Reply by GiudGiud** [reply]
Hello

time is `t` in parsed expressions. 

is `1.25 * * 270` what you wanted to have? Does not seem correct

Guillaume
